tools/lou_checkyaml.c

```diff
@@ -668,20 +735,34 @@ main(int argc, char *argv[]) {
 	}
 	yaml_event_delete(&event);
 
-	if (!yaml_parser_parse(&parser, &event))
-		simple_error("table expected", &parser, &event);
+	int has_next;
+	has_next = yaml_parser_parse(&parser, &event);
+
+	const char *display_table = NULL;
```

Comment:
I don't quite understand this. What is the YAML syntax here? Where do you define a display table? And why do you need this? Can't you just define the display table with the table? Ah, maybe you can't when doing a table query. Is that it?

Ah, never mind I can see it in the YAML files.

Reply:
Yes, that's it. And a second reason is that I want to have display and translation tables separated in the code (https://github.com/liblouis/liblouis/issues/307). Separating them in the tests is already a first step.